sdk/attestation/azure-security-attestation/inc/azure/attestation/attestation_client.hpp

```diff
@@ -114,6 +114,9 @@ namespace Azure { namespace Security { namespace Attestation {
    */
 
   class AttestationClient final {
```

Comment:
This class also doesn't technically need a virtual destructor. But as above, I cannot see how it can hurt in practice, and if choosing between two, virtual is better.